src/scss/components/infinite_table.scss

```diff
@@ -5,10 +5,13 @@
 	    flex-basis: 80% !important;
 	}
 
-	&.ui.basic.primary.button {
+	&.ui.primary.basic {
 	    box-shadow: 0px 0px 0px 0px white inset !important;
 	    font-weight: bold !important;
-	    text-align: center;
+		text-align: center;
+		color: #1E5AA8 !important;
+		cursor: pointer;
+		user-select: none;

```

Comment:
And the user select? These two only show in buttons, and you've correctly removed the button class here